### PR TITLE
Add tp-logfile replay functionality, defaults to true

### PR DESCRIPTION
**Pull request by AquaqProdSupport1**
vtwapsubscriber now replays current day logfile, before appending to tables for real-time data from TP.

### CODE REVIEW

File: code/processes/vtwap_subscriber.q [hunk 1]
```diff
@@ -20,26 +25,28 @@ notpconnected:{[]
 
 \d .
 
-.servers.CONNECTIONS:distinct .servers.CONNECTIONS,.vtwap.tickerplanttypes;                      //make sure that the process will make a connection to any process of rdb type
+.servers.CONNECTIONS:distinct .servers.CONNECTIONS,.vtwap.tickerplanttypes;                     //make sure that the process will make a connection to any process of rdb type
 .lg.o[`init;"searching for servers"];                                                           //and append connection results to logfile
 .servers.startup[];
-.vtwap.subscribe[]                                                                               //subscribe to the rdb
+.vtwap.subscribe[]                                                                              //subscribe to the rdb
```
Comment (ThomasSmyth) [inline code/processes/vtwap_subscriber.q]: Semi colon.

File: code/processes/vtwap_subscriber.q [hunk 2]
```diff
@@ -11,7 +16,7 @@ if[not .timer.enabled;.lg.e[`symcheckinit;
 subscribe:{[]
   if[0=count s:.sub.getsubscriptionhandles[tickerplanttypes;();()!()];:()];
   .lg.o[`subscribe;"found available tickerplant, attempting to subscribe"];
-  subinfo:.sub.subscribe[subscribeto;subscribesyms;schema;0b;first s];
+  subinfo:.sub.subscribe[subscribeto;subscribesyms;schema;replaylog;first s];
   @[`.vtwap;;:;][key subinfo;value subinfo];
```
Comment (ThomasSmyth) [inline code/processes/vtwap_subscriber.q]: Can skip the projection:``@[`.vtwap;key subinfo;:;value subinfo]``, it's whats happening anyway.

File: code/processes/vtwap_subscriber.q [hunk 3]
```diff
@@ -1,6 +1,11 @@
-\d .vtwap                                                                             //get rdb info and schema, reconnection wait time
+upd:{[t;x]                                                                                      //set upd function for log replay
+  if[not t in `trade`trade_iex`quote`quote_iex;:()];
```
Comment (ThomasSmyth) [inline code/processes/vtwap_subscriber.q]: Feel bad commenting on this after pointing these out last time but in this case you could just have `if[...;t insert x]`. For me this is short enough to be a one liner, in those other case it made sense to reverse the condition as there was many actions to undertake.

In saying that I don't mind if you leave this as-is now.

File: code/processes/vtwap_subscriber.q (hunk 2 above)
Comment (AquaqProdSupport1) [reply code/processes/vtwap_subscriber.q]: This is present in other TorQ processes, like the rdb. Does it make sense to keep this as close to like-schemas as possible, or is it still superfluous?

File: code/processes/vtwap_subscriber.q (hunk 3 above)
Comment (AquaqProdSupport1) [reply code/processes/vtwap_subscriber.q]: I think it's a fair-enough change to make.

File: code/processes/vtwap_subscriber.q (hunk 2 above)
Comment (ThomasSmyth) [reply code/processes/vtwap_subscriber.q]: I think just change it for neatness, I don't think that would be consistency worth keeping.

**Pull request merged**